Process and Policy Pages for Submitting Data (#1627)
agrc/gis.utah.gov: sharing/core-data/index.md

| Realtime GPS Correction Service | Highway Milepost Reference | Highway Exit Numbers and Names |
| State Agency Data Layers in SGID (DNR, DEQ, SITLA) | USGS Topo Quad Web Services | State Cadastral Grant Funding |

> GIS data can be shared to AGRC for publication in the State Geographic Information Database (and related web serves) via the [SGID Open Data site]({% link about/policy/open-data/index.md %}), [email]({% link about/contact/index.html %}), [Dropbox](https://www.dropbox.com/), [ShareFile](https://www.sharefile.com/), or we can pull from an ArcSDE connection or web-accessible file geodatabase or shapefile (https://, ftp://, etc.).

Reviewer: spelling and more pulling places

```suggestion
GIS data can be shared to AGRC for publication in the State Geographic Information Database (and related web services) via the [SGID Open Data site]({% link about/policy/open-data/index.md %}), [email]({% link about/contact/index.html %}), [Dropbox](https://www.dropbox.com/), [ShareFile](https://www.sharefile.com/), or we can pull from a database connection, web-accessible map service, feature service, file geodatabase, or shapefile (https://, ftp://, etc.).
```

Author: Whoops, this page should have been removed. It's being replaced wholesale by /sharing and /sharing/process.
